File: src/main/resources/output/IFC4X3_java/Entities/IfcRoot/IfcPropertyDefinition/IfcPropertySetDefinition/IfcPreDefinedPropertySet/IfcWindowPanelProperties/IfcWindowPanelProperties.java
public class IfcWindowPanelProperties extends IfcPreDefinedPropertySet {
    public IfcWindowPanelOperationEnum OperationType;
    public IfcWindowPanelPositionEnum PanelPosition;
    public IfcPositiveLengthMeasure FrameDepth;
    public IfcPositiveLengthMeasure FrameThickness;
    public IfcShapeAspect ShapeAspectStyle;

    // === WHERE CLAUSES ===
    // ApplicableToType : (EXISTS(SELF\IfcPropertySetDefinition.DefinesType[1])) AND ('IFC4X3_DEV_73740fe4.IFCWINDOWTYPE' IN TYPEOF(SELF\IfcPropertySetDefinition.DefinesType[1]))
}

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcPropertyDefinition/IfcPropertySetDefinition/IfcPreDefinedPropertySet/IfcWindowPanelProperties/IfcWindowPanelProperties.cs
public class IfcWindowPanelProperties : IfcPreDefinedPropertySet
{
    public IfcWindowPanelOperationEnum OperationType { get; set; }
    public IfcWindowPanelPositionEnum PanelPosition { get; set; }
    public IfcPositiveLengthMeasure FrameDepth { get; set; }
    public IfcPositiveLengthMeasure FrameThickness { get; set; }
    public IfcShapeAspect ShapeAspectStyle { get; set; }

    // === WHERE CLAUSES ===
    // ApplicableToType : (EXISTS(SELF\IfcPropertySetDefinition.DefinesType[1])) AND ('IFC4X3_DEV_73740fe4.IFCWINDOWTYPE' IN TYPEOF(SELF\IfcPropertySetDefinition.DefinesType[1]))
}
